src/components/UserProfile/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from "react";
+import { useSelector } from "react-redux";
+import "./style.css";
+
+export function UserCont() {
+
+    const user = useSelector((state) => state.result[0]);
+    const error = useSelector((state) => state.error);
+
+    const [userData, setUserData] = useState();
+
+    useEffect(() => {
+        const renderUser = () => {
+    
+          setUserData(
+            <section aria-label="user-info" className="userInfoCont">
+              <div className="user-info">
+                <span role="userLogin" className="user-login">{user.owner.login}</span>
+                <br />
+                {/* <span role="userName" className="user-name">{user.name}</span> */}
+              </div>
+    
+              <img className="profile-img" src={user.owner.avatar_url} alt="profile picture"/>
+    
+              <div className="user-bio">
+                <p role="user-bio">{user.bio}</p>
+              </div>
+            </section>
+          )
+        };
+        renderUser()
+    }, [user])
+    
+    return (
+        <>
+          { error ? <p role="alert">Could not find user data</p> : userData }
```

Comment:
From typing in a false name, this never gets called: it tries to reach /user/repos first and gets a 404.